cirq-ionq/cirq_ionq/ionq_client_test.py

```diff
@@ -246,6 +246,25 @@ def test_ionq_client_create_job_timeout(mock_post):
         )
 
 
+@mock.patch('requests.get')
+def test_ionq_client_get_job_retry_409(mock_get):
+    response1 = mock.MagicMock()
+    response2 = mock.MagicMock()
+    mock_get.side_effect = [response1, response2]
+    response1.ok = False
+    response1.status_code = requests.codes.conflict
+    response1.request.method = "GET"
+    response2.ok = True
+    response2.json.return_value = {'foo': 'bar'}
+
+    client = ionq.ionq_client._IonQClient(remote_host='http://example.com', api_key='to_my_heart')
```

Comment:
@dabacon @Cynocracy I still love the dummy API key we use for these tests

Reply:
lol, it's pretty great